constants/Items.json

```diff
@@ -59,5 +59,50 @@
     "quantum": 1,
     "ultimate_the_one": 1,
     "rainbow": 1
+  },
+  "value_calculation_data": {
+    "always_active_enchants": {
+      "scavenger": {
+        "level": 5,
+        "items": [
+          "CRYPT_DREADLORD_SWORD",
+          "ZOMBIE_SOLDIER_CUTLASS",
+          "CONJURING_SWORD",
+          "EARTH_SHARD",
+          "ZOMBIE_KNIGHT_SWORD",
+          "SILENT_DEATH",
+          "ZOMBIE_COMMANDER_WHIP",
+          "ICE_SPRAY_WAND"
+        ]
+      },
+      "replenish": {
+        "level": 1,
+        "items": [
+          "ADVANCED_GARDENING_HOE",
+          "ADVANCED_GARDENING_AXE"
+        ]
+      }
+    },
+    "tiered_enchants": [
+      "compact",
+      "cultivating",
+      "champion",
+      "expertise",
+      "hecatomb",
+      "toxophilite"
+    ],
```

Comment:
these are already in another file, idk if we want to duplicate them or just use the data from the other file

Reply:
Might as well keep the more informative of the two. I'll rip it out of here and use StackingEnchants over there.